src/panels/lovelace/editor/hui-entities-card-row-editor.ts

```diff
@@ -0,0 +1,271 @@
+import { mdiClose, mdiDrag } from "@mdi/js";
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+import { guard } from "lit-html/directives/guard";
+import type { SortableEvent } from "sortablejs";
+import Sortable, {
+  AutoScroll,
+  OnSpill,
+} from "sortablejs/modular/sortable.core.esm";
+import { fireEvent } from "../../../common/dom/fire_event";
+import "../../../components/entity/ha-entity-picker";
+import type { HaEntityPicker } from "../../../components/entity/ha-entity-picker";
+import "../../../components/ha-icon-button";
+import { sortableStyles } from "../../../resources/ha-sortable-style";
+import { HomeAssistant } from "../../../types";
+import { EntityConfig, LovelaceRowConfig } from "../entity-rows/types";
+
+@customElement("hui-entities-card-row-editor")
+export class HuiEntitiesCardRowEditor extends LitElement {
+  @property({ attribute: false }) protected hass?: HomeAssistant;
+
+  @property({ attribute: false }) protected entities?: LovelaceRowConfig[];
+
+  @property() protected label?: string;
+
+  @internalProperty() private _attached = false;
+
+  @internalProperty() private _renderEmptySortable = false;
+
+  private _sortable?: Sortable;
+
+  public connectedCallback() {
+    super.connectedCallback();
+    this._attached = true;
+  }
+
+  public disconnectedCallback() {
+    super.disconnectedCallback();
+    this._attached = false;
+  }
+
+  protected render(): TemplateResult {
+    if (!this.entities || !this.hass) {
+      return html``;
+    }
+
+    return html`
+      <h3>
+        ${this.label ||
+        `${this.hass!.localize(
+          "ui.panel.lovelace.editor.card.generic.entities"
+        )} (${this.hass!.localize(
+          "ui.panel.lovelace.editor.card.config.required"
+        )})`}
+      </h3>
+      <div class="entities">
+        ${guard([this.entities, this._renderEmptySortable], () =>
+          this._renderEmptySortable
+            ? ""
+            : this.entities!.map((entityConf, index) => {
+                return html`
+                  <div class="entity">
+                    <ha-svg-icon class="handle" .path=${mdiDrag}></ha-svg-icon>
+                    ${entityConf.type
+                      ? html`
+                          <div class="special-row">
+                            <div>
+                              <span>
+                                ${this.hass!.localize(
+                                  `ui.panel.lovelace.editor.card.entities.entity_row.${entityConf.type}`
+                                )}
+                                ${this.hass!.localize(
+                                  "ui.panel.lovelace.editor.card.entities.special_row"
+                                )}
```

Comment:
We should combine this in 1 localize:
```suggestion
                                ${this.hass!.localize(
                                  "ui.panel.lovelace.editor.card.entities.special_row", "row_type", this.hass!.localize(
                                  `ui.panel.lovelace.editor.card.entities.entity_row.${entityConf.type}`
                                )
                                )}
```

But maybe we should just drop the `special_row`? Just "Button row" would be enough?